File: Python Excercise/Part_1.py
# Part 1: Password Validator
# A user creates an account by entering their password twice. Check if they match.

# Write a Python program that:

# Takes two strings as input (password and confirmation)
# Prints the length of the first string
# Prints the length of the second string
# Prints if the lengths match
# Prints if the two strings are the same
# Example:

# Input: "movie@2026" and "movie@2026"
# Output:
# Length 1: 10
# Length 2: 10
# Lengths match: True
# Strings match: True
# Part 1: Password Validator

password = input("Enter password: ")
confirm_password = input("Confirm password: ")

# Lengths
len1 = len(password)
len2 = len(confirm_password)

print("Length 1:", len1)
print("Length 2:", len2)

# Compare lengths
print("Lengths match:", len1 == len2)

# Compare strings
print("Strings match:", password == confirm_password)

File: javaex/Part_1.java
// Part 1: Username Checker
// A fan registers for tickets by entering their username twice. Check if they match.

// Write a Java program that:

// Takes two strings as input (username and confirmation)
// Prints the length of the first string
// Prints the length of the second string
// Prints if the lengths match
// Prints if the two strings are the same
// Example:

// Input: "kduRocker2026" and "kduRocker2026"
// Output:
// Length 1: 11
// Length 2: 11
// Lengths match: true
// Strings match: true

import java.util.Scanner;

public class Part_1{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        System.out.print("Enter username: ");
        String username = sc.nextLine();

        System.out.print("Confirm username: ");
        String confirmation = sc.nextLine();;

        System.out.println("Length 1: " + username.length());
        System.out.println("Length 2: " + confirmation.length());
        System.out.println("Lengths match: " + (username.length() == confirmation.length()));
        System.out.println("Strings match: " + username.equals(confirmation));
        sc.close();
    }
}
